### PR TITLE
Use summary type to observe p99

### PR DESCRIPTION
**Pull request by jyz0309**
Signed-off-by: jyz0309 <45495947@qq.com>

**What this PR does**: 
Change metric type to observe RT P99 value.
**Which issue(s) this PR fixes**: 
None.

**You should pay attention to items below to ensure your pr passes our ci test**
We do not merge pr with ci tests failed

- [x] All ut passed (run 'go test ./...' in project root)
- [x] After go-fmt ed , run 'go fmt project' using goland.
- [x] Golangci-lint passed, run 'sudo golangci-lint run' in project root.
- [x] After import formatted, (using [imports-formatter](https://github.com/dubbogo/tools#5-how-to-get-imports-formatter) to run 'imports-formatter .' in project root, to format your import blocks, mentioned in [CONTRIBUTING.md](https://github.com/apache/dubbo-go/blob/master/CONTRIBUTING.md) above) 
- [x] Your new-created file needs to have [apache license](https://raw.githubusercontent.com/dubbogo/resources/master/tools/license/license.txt) at the top, like other existed file does.
- [x] All integration test passed. You can run integration test locally (with docker env). Clone our [dubbo-go-samples](https://github.com/apache/dubbo-go-samples) project and replace the go.mod to your dubbo-go, and run 'sudo sh start_integration_test.sh' at root of samples project root. (M1 Slice is not Support)

### CODE REVIEW

File: metrics/prometheus/reporter.go [hunk 1]
```diff
@@ -78,9 +78,9 @@ func init() {
 // https://prometheus.io/docs/guides/go-application/
 type PrometheusReporter struct {
 	// report the consumer-side's rt gauge data
-	consumerRTGaugeVec *prometheus.GaugeVec
+	consumerRTGaugeVec *prometheus.SummaryVec
```
Comment (pherzheyu) [inline metrics/prometheus/reporter.go:81]: consumerRTGaugeVec名称修改下or新增一个新的字段？

Comment (jyz0309) [reply metrics/prometheus/reporter.go]: Done. PTAL again.

**Review by LaurenceLiZhixin — APPROVED** [review]

**Comment by LaurenceLiZhixin** [comment]
Good， and it's better to add Max Age configuration at 
https://github.com/apache/dubbo-go/pull/1875/files#diff-dabf5562f4d32b8bc69d22e50f651690c48257c90f56f7104cde9fc014af1cd3R217

and export the configuration to dubbogo.yaml , metrics config struct of root config.

**Comment by jyz0309** [comment]
> Good， and it's better to add Max Age configuration at https://github.com/apache/dubbo-go/pull/1875/files#diff-dabf5562f4d32b8bc69d22e50f651690c48257c90f56f7104cde9fc014af1cd3R217
> 
> The sdk: https://github.com/prometheus/client_golang/blob/fa4aa9000d2863904891d193dea354d23f3d712a/prometheus/summary.go#L207
> 
> and export the configuration to dubbogo.yaml , metrics config struct of root config.
> 
> @jyz0309

Can you tell me how to export configuration to dubbogo.yaml? It will help a lot if there is a sample code. @LaurenceLiZhixin

**Comment by LaurenceLiZhixin** [comment]
Add a config field named "RTSummaryMaxAge" of metrics config here: https://github.com/apache/dubbo-go/blob/master/config/metric_config.go#L31
And set this config field to your summary.
@jyz0309

**Review by AlexStocks — APPROVED** [review]

**Review by justxuewei — APPROVED** [review]

**Review by pherzheyu — APPROVED** [review]

**Pull request merged**